docs/files.md

```diff
@@ -113,6 +114,14 @@ method will move the file to the user's trash.
 ```js
 client.files.delete('12345', callback);
 ```
+Get a Trashed File
+---------------
+
+Information about a file in the trash can be retrieved with the [`files.getTrashedFile(fileID, callback)`](http://opensource.box.com/box-node-sdk/Folders.html#getTrashedFile) method.
```

Comment:
The URL to the docs should be http://opensource.box.com/box-node-sdk/Files.html#getTrashedFile